src/main/main.c

```diff
@@ -1350,12 +1353,15 @@ m64p_error main_run(void)
     if (count_per_op <= 0)
         count_per_op = ROM_PARAMS.countperop;
 
+    if (count_per_op_denom_pot > 50)
```

Comment:
I think this is way too big of a number for the power-of-two denominator.  Maybe 8 makes sense? When would you ever need to overclock by more than 256x?

Reply:
Yeah, you're right - anything over 10 is unnoticeable and more likely to cause issues. I'll set to 8 then.